File: Factorial/Factorial.cpp
#include <iostream>

using namespace std;

unsigned int factorial(unsigned int n)
{
    if (n == 0)
        return 1;
    return n * factorial(n - 1);
}

// Driver code
int main(){
     cout << "Factorial : ( n! = 1 * 2 * 3 * ... * n ) \n\n"
         << "For example : ( 5! = 1 * 2 * 3 * 4 * 5 = 120 ) \n"
         << "-------------------------------------------------\n";
    int num;
    cout << "Enter the number (n) : ";
    cin  >> num;

    cout << "Factorial of "
         << num << " is \n{" << factorial(num) << "}" << endl;
    return 0;
}

File: factorial/Factorial.py
import sys


def factorial(n):
    """
        using recursive function
        this function take an number and return factorial value of it         
    """
    if n == 1:
        return 1
    
    return factorial(n-1) * n


def main():
    n = input("Enter a Number: ")
    try:
        n = int(n)
    except ValueError:
        sys.exit("Input Must be Integer")

    print(f"Factorial Value if {n} is: {factorial(n)}")



if __name__ == '__main__':
    main()
